docs/asciidoc/migration/v5-to-v6.adoc

```diff
@@ -28,6 +28,14 @@ Dozer is built against Java JDK 1.8 ONLY.
 === 3. dozer-osgi dropped
 You no longer need to use the below, if you are running in an OSGi environment:
 
+=== 4. Artifacts renamed
```

Comment:
this migration doc should not be changed. its only for people moving from 5.x to 6.0. please revert

Reply:
Done.